README.md

```diff
@@ -115,7 +115,7 @@ The custom certificates must also be placed into the trusted directory so that a
 * `/docker/config/certs`
 
 ## Import trusted SSL certificates
-If you want your application to trust certain, possibly self-signed, SSL certificates, put them in PEM format in the following directory on your Docker host: 
+If you want your application to trust certain, possibly self-signed, SSL certificates, put them (e.g. your vSphere certificate) in PEM format in the following directory on your Docker host: 
```

Comment:
@gertipoppel Could you take a look and possibly tell how to update it? I think it is important to point that here.

Reply:
@maciaszczykm Hi, together with Frank, we have extended the whole two sections on certificates. Could you check again and replace the following in the Readme file: 

## Import SSL keypairs for the application listeners
If you want to use your own SSL key pairs that your application is to use, replace the default key pair by your PEM files in the following directories on your Docker host: 

* Private key: `/docker/config/<CONTAINER_NAME>/ssl/privkey`
* Public certificate: `/docker/config/<CONTAINER_NAME>/ssl/cert`
* Intermediates / chain (optional): `/docker/config/<CONTAINER_NAME>/ssl/chain`

Note:

Replace `/docker` with the directory where Docker is installed, and `<CONTAINER_NAME>` with the respective container name, e.g. `oscm-core`.

The custom certificates must also be placed into the trusted directory so that a trusted relationship between the containers is established: 

* `/docker/config/certs`

This directory is shared by all containers. By default, if you use your own SSL key pairs, you must also place all the public certificate files here.

For example, if you have a custom SSL keypair for the `oscm-core` container, you need to place the private key into the `/docker/config/oscm-core/ssl/privkey` directory, and the public certificate into the `/docker/config/oscm-core/ssl/cert` directory. Additionally, you need to place the public certificate into the `/docker/config/certs` directory on your Docker host. In this case, a restart of the `oscm-core` and `oscm-app` containers is required.

## Import trusted SSL certificates
If you want your application to trust certain, possibly self-signed, SSL certificates, put them (e.g. your vSphere certificate) in PEM format in the following directory on your Docker host: 

* `/docker/config/certs`

Note:

Replace `/docker` with the directory where Docker is installed. 

For example, if you want to use the VMware service controller, you need to export the vSphere certificate in PEM format, and copy it to the `/docker/config/certs` directory.  Since the VMware service controller is running in the `oscm-app` container, a restart of this container is required.

Reply:
@maciaszczykm I mean, I could also do it :-)

Reply:
@maciaszczykm 
the change looks good for me. Just one question. Where is are the controller properties initialized?
Following section is commented https://github.com/servicecatalog/oscm-dockerbuild/pull/117/commits/6bdc26d35bf78f2d8f327e4b17006e5a2f9fed0e#diff-94e3e883fea0e82eee2fb61d8eb50303R246
If this isn't needed, please remove or replace it.

Reply:
@gertipoppel You can commit directly to branch with this pull request or create separate one. I can also copy your changes, but I need your confirmation.

@GoebelL We are importing it with other VMware sample data, that is always deployed:
https://github.com/servicecatalog/oscm-dockerbuild/pull/117/files#diff-3e7c039bbfdfe20d9fdc3d9fe7b93604R3

It was commented in one of the previous commits, please check current state of the file: https://github.com/servicecatalog/oscm-dockerbuild/pull/117/files#diff-94e3e883fea0e82eee2fb61d8eb50303R242

Reply:
@maciaszczykm Please just copy to your own PR. Thx

Reply:
Done :)

Reply:
I'll be happy to merge this to master, after you have resolved the [merge conflict](https://github.com/servicecatalog/oscm-dockerbuild/pull/117/conflicts).